pyproject.toml

```diff
@@ -23,7 +23,8 @@ classifiers = [
 
 requires-python = ">=3.8"
 dependencies = [
-    "httpx >= 0.24.1"
+    "apify-shared ~= 1.0.0b2",
```

Comment:
Yeah I was writing that too, I wanted to wait until we test that the beta version works, and then publish the stable one, but since the tests have passed here it should be fine

Reply:
Ok, I released `apify_shared` `v1.0.0` and use it.